index.js

```diff
@@ -1530,12 +1534,23 @@ class Command extends EventEmitter {
         return passthru;
       };
     }
+    globalEventEmitter.emit(this._helpHeader, this);
+    this.emit(this._helpHeader, this);
     const cbOutput = cb(this.helpInformation());
     if (typeof cbOutput !== 'string' && !Buffer.isBuffer(cbOutput)) {
       throw new Error('outputHelp callback must return a string or a Buffer');
     }
-    process.stdout.write(cbOutput);
-    this.emit(this._helpLongFlag);
+    const globalBodyListeners = globalEventEmitter.listeners(this._helpBody);
+    const bodyListeners = this.listeners(this._helpBody);
```

Comment:
I have been moving to a different pattern for testing listeners, since we don't call the listener directly we do not need the list. Just check the count.

```js
if (this.listenerCount('help:body')) {
   this.emit('help:body', this);
} else {
   process.stdout.write(cbOutput);
}
```

Reply:
How is `this.litenerCount` supposed to know how many listeners there are? Or is this just an indirect call to `this.listeners('whatever').length`?

Reply:
`listenerCount` is an EventEmitter method: https://nodejs.org/api/events.html#events_emitter_listenercount_eventname

Reply:
Oops. Yeah is missed that. 😅